dialects/linux/dproc.c

```diff
@@ -1518,12 +1574,91 @@ process_proc_map(p, s, ss)
 		efs = sv = 1;
 	    else
 		efs = 0;
-	    if (!efs) {
-		if (HasNFS)
-		    sv = statsafely(fp[6], &sb);
-		else
-		    sv = stat(fp[6], &sb);
+
+	    /* kernel version >= 3.8.0 and the target process has a different
+	     * mount namespace with this lsof process. */
+	    if (compare_kernel_release(38000) && compare_namespace("mnt")) {
```

Comment:
Do we have to do this every time when extracting information from a mmap entry?
I think we should do this ONLY once with RUNTIME information.
uname(2) is for getting runtime information.

Reply:
Will do this before the while loop.